### DISCUSSION TITLE
Displaying to which server RestDesk is actually connected

### DISCUSSION THREAD

**Discussion by pedjas** [rustdesk/rustdesk#3826] [General]
When client is connected to server it is hard to find out to which server it connected to. Actually, the only place to check is About dialog, but it seems it shows configured server but not the actual server it is connected to.

In other bug report I pointed out issue when there are two or more RestDesk clients set to connect to different sever, as they interfere each other. One of the issues is user cannot see which server client is connected to. This information should be easily visible, maybe in caption of the client window or next to "Ready" status message displayed at the bottom after connection is achieved. Actually, it should show server info even on failed connection.

It should show exact info to which server connection is made, not just what is configured. Connection port info would also be helpful.

**Comment by mrwrighty** [comment]
Why would you need to know which server a client is connected to. What difference does it make to the client as long as you can connect to it. You can only have one server configured on the client anyway.

**Comment by pedjas** [comment]
One may have multiple RustDesk standalone exes configured to connect to different servers.
When run they are all the same. There is no visual indication which one is running.

**Comment by ZuratonE** [comment]
I would also suggest to add more than one server configuration.
For example, I have two selfhosted servers: one at home and other at work. And in that case I would like to select to which one I want to connect/login.

**Comment by tracure1337** [comment]
This is needed yes :)
